PowerShell Scanners/Get-ChocolateyPackages/Scan Profile.xml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<AdminArsenal.Export Code="PDQInventory" Name="PDQ Inventory" Version="19.0.0.0" MinimumVersion="19.0">
+  <ScanProfile>
+    <Collections type="list" />
+    <Scanners type="list">
+      <Scanner>
+        <ModifiedDate>2020-04-28T11:57:58.0000000-06:00</ModifiedDate>
+        <Name>Chocolatey Package Information</Name>
+        <Script></Script>
+        <FileName>C:\PowerShell-Scanners\PowerShell Scanners\Get Chocolatey Packages\Get-ChocolateyPackageData.ps1</FileName>
```

Comment:
Isn't your folder name `Get-ChocolateyPackages`, not `Get Chocolatey Packages`?

Reply:
That was me being heavy handed. I've fixed this.